packages/open-payments/src/client/incoming-payment.test.ts

```diff
@@ -13,10 +14,22 @@ import {
   defaultAxiosInstance,
   mockILPStreamConnection,
   mockIncomingPayment,
+  mockIncomingPaymentPaginationResult,
   mockOpenApiResponseValidators,
   silentLogger
 } from '../test/helpers'
 import nock from 'nock'
+import { v4 as uuid } from 'uuid'
+import * as requestors from './requests'
+import { getRSPath } from '../types'
+
+jest.mock('./requests', () => {
+  return {
+    // https://jestjs.io/docs/jest-object#jestmockmodulename-factory-options
+    __esModule: true,
+    ...jest.requireActual('./requests')
```

Comment:
I think we can mock out the `get` here, so we don't need to `.mockResolvedValueOnce(incomingPaymentPaginationResult)` on the `getSpy` below

Reply:
I also think we can remove `__esModule: true,` because we aren't using a default export

Reply:
> I think we can mock out the `get` here, so we don't need to `.mockResolvedValueOnce(incomingPaymentPaginationResult)` on the `getSpy` below

If we mock the `get`, all the tests that are using it will fail:
```
Test Suites: 1 failed, 1 total
Tests:       8 failed, 26 passed, 34 total
```
<img width="962" alt="image" src="https://user-images.githubusercontent.com/60383558/208117218-7e7aca45-9c7d-4893-b86b-afae069c5f5e.png">

---
> I also think we can remove __esModule: true, because we aren't using a default export

When `__esModule: true` is not specified, we get the following error:

```
incoming-payment › routes › list › calls get method with correct validator

    Error when making Open Payments GET request: Nock: No match for request {
      "method": "GET",
      "url": "http://localhost:1000/.well-known/pay/incoming-payments",
      "headers": {
        "accept": "application/json, text/plain, */*",
        "content-type": "application/json",
        "authorization": "GNAP accessToken",
        "signature": "sig1=:VYmcFqf6cSzLN10+dG/TnXQI23FxWY/n2/wVbNW4oX+U4rCBHpbT75AaMZOe3qzoUDiQUxZg0fpXjSBxHkNpDw==:",
        "signature-input": "sig1=(\"@method\" \"@target-uri\" \"authorization\");created=1671199508;keyid=\"default-key-id\";alg=\"ed25519\"",
        "user-agent": "axios/1.1.2",
        "accept-encoding": "gzip, deflate, br"
      }
    }
```